simpleSequential/unionTwoSet.py

```diff
@@ -3,3 +3,5 @@
 #   unionTwoSet({2, 3}, {3, 2}) # Return a set that equal to {2, 3}
 #   unionTwoSet({2, 3}, {4, 3}) # Return a set that equal to {2, 3, 4}
 #   unionTwoSet({2, 3}, {5, 4}) # Return a set that equal to {2, 3, 4, 5}
+def unionTwoSet(firstSet: set, secondSet: set) -> set:
```

Comment:
1/ Let use's "intSet1" and "intSet2" instead of "firstSet" and "secondSet". The amount of characters is less but more meaningful, and covers the actual type which is a "set of integers".

2a/ Did you import "Set" from typing?

2b/ Please specify this type as a "set of integers".